packages/admin/cms-admin/src/userPermissions/UserGrid.tsx

```diff
@@ -52,13 +60,45 @@ export const UserPermissionsUserGrid = ({ toolbarAction, rowAction, actionsColum
             headerName: "",
             sortable: false,
             filterable: false,
-            type: "actions",
+
```

Comment:
Why remove the "actions" type? This column is still used for actions, like "Edit" and "Impersonate".